contracts/README.md

```diff
@@ -1,16 +1,17 @@
-## Teleporter Messenger Contract
-This directory contains Solidity contracts implementing the Teleporter messaging protocol.
+## Avalanche Warp Messaging Contracts
+This directory contains Solidity contracts that utilize Avalanche Warp Messaging for a variety of cross-chain workflows. This includes:
+- Governance contracts for authenticating messages directly from validators using Warp
+- Mock contracts used for testing
+- Teleporter contracts for sending cross-chain messages using the Teleporter protocol
+- General purpose utilities that are used by the above applications
 
-This directory is set up as a [Foundry](https://github.com/foundry-rs/foundry) project. Use the `scripts/install_foundry.sh` to install the correct version of the ava-labs fork of foundry. Further documentation about given contracts can be found in`src/teleporter/`.
+This directory is set up as a [Foundry](https://github.com/foundry-rs/foundry) project. Use the `scripts/install_foundry.sh` to install the correct version of the ava-labs fork of foundry. Further documentation about given contracts can be found in`contracts/teleporter/`.
 
 ## Building and Running
 - To compile the contracts run `forge build` from this directory.
 - Similarly, to run unit tests, run `forge test`.
 - See additional testing and deployment options [here](https://book.getfoundry.sh/forge/).
 
-## Deployment
```

Comment:
Do we still want to include this in the teleporter readme? It's a common pitfall and having a **bold** warning might still be good on top of the Deployment section there

Reply:
IMO the existing warnings are sifficient. There's a note in both the top-level README, as well as a section in `contracts/teleporter/README.md`